app/assets/stylesheets/newflow_layout.scss

```diff
@@ -18,82 +44,346 @@ body {
 #signinup-form {
     max-width: 75rem;
     border-top: 0;
-    margin-top: 3rem;
     margin-left: auto;
     margin-right: auto;
+    margin-bottom: 3rem;
+    margin-top: 4rem;
+    padding-left: 14px;
+    padding-right: 14px;
+
+    .step-counter {
+        height: 30px;
+
+        > div {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 94px;
+            height: 100%;
+
+            background-color: #f5f5f5;
+            border-radius: 1.5rem;
+        }
+
+        span {
+            font-size: 1.4rem;
+            font-weight: bold;
+            letter-spacing: -0.56px;
+            // padding: 1rem;
+        }
+    }
+
+    .tab-group {
+        padding: 0;
+
+        .tab {
+            @include width-up-to($phone-max) {
+                width: 50%;
+            }
+
+            width: 18rem;
+            font-size: 1.8rem;
```

Comment:
The pattern library has font macros. You could do
`@include set-font(h4)` to get bold 1.8rem sizing with the standard slightly-narrowed-spacing that we use for titles. Or, equivalently, `@include title-font(1.8rem)`.

Reply:
Done.